### PR TITLE
fix: click events aren't triggering for line charts

### PR DESCRIPTION
**Pull request by jordanstith15**
Closes [COMUI-2307](https://inindca.atlassian.net/browse/COMUI-2307)

[COMUI-2307]: https://inindca.atlassian.net/browse/COMUI-2307?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx [hunk 1]
```diff
@@ -66,12 +66,7 @@ export class GuxVisualization {
         visSpec.signals.push({
           name: 'chartClick',
           value: 0,
-          on: [
-            { events: 'rect:mousedown', update: 'datum' },
-            { events: 'arc:mousedown', update: 'datum' },
-            { events: 'line:mousedown', update: 'datum' },
-            { events: 'symbol:mousedown', update: 'datum' }
-          ]
+          on: [{ events: '*:mousedown', update: 'datum' }]
```
Comment (jason-evans-genesys) [inline packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx:69]: I noticed that `gux-chart-column` logs when the chart is clicked with this line `chart1.addEventListener('chartClicked', (event) => console.info(event));` - do you think it's worth adding in `gux-visualization` to demonstrate this chart click event firing properly on the example page?

Comment (jordanstith15) [reply packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx:69]: is that different than the log that already shows on the example page when you click?

Comment (jason-evans-genesys) [reply packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx:69]: I'm not seeing a log on the example page for `gux-visualization`, but I do see a log shows on the example page for `gux-chart-column`

Comment (jordanstith15) [reply packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx:69]: Oh I mean the notification. Not sure if it's worth having the log if we already show a notification for the chartClicked event

Comment (jason-evans-genesys) [reply packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx:69]: Agreed. I didn't notice the notification at first, but that's good as is.

**Review by jason-evans-genesys — APPROVED** [review]

**Pull request merged**